### ISSUE TITLE
Glyph on U+26C4 should not be here

### ISSUE THREAD

**Issue by delphinus** [JetBrains/JetBrainsMono#39]
v1.0 has a strange glyph on U+26C4 (SNOWMAN WITHOUT SNOW).

<img width="532" alt="スクリーンショット 0002-01-17 16 42 55" src="https://user-images.githubusercontent.com/1239245/72594596-16702480-394b-11ea-8d0c-4bf3121f788d.png">

This has two problems.

1. Unicode defines [East Asian Width][eaw]. [The spec txt][tst] describes that this codepoint (U+26C4) is “W”, that is, “Wide Character”.

   ```
   ...
   26C4..26C5;W     # So     [2] SNOWMAN WITHOUT SNOW..SUN BEHIND CLOUD
   ...
   ```

   Terminal apps, such as Terminal.app and iTerm2, respect this value and draw U+26C4 as a double-width one. So it shows an extra space. It is annoying.

   <img width="189" alt="スクリーンショット 0002-01-17 17 23 49" src="https://user-images.githubusercontent.com/1239245/72595868-25a4a180-394e-11ea-84d4-1460b8d1f097.png">

2. This codepoint is NOT the Private Use Area. We should not use here for this. I suggest we move this glyph to PUA (one is U+E000 .. U+F8FF). This also solves 1.

[eaw]: http://www.unicode.org/reports/tr11/
[tst]: http://www.unicode.org/Public/UCD/latest/ucd/EastAsianWidth.txt

**Comment by philippnurullin** [comment]
Ok, we will move our version owl to new place. 
Thanks for feedback!

**Comment by philippnurullin** [comment]
Fixed in 1.0.1
